### ISSUE TITLE
Update plan?

### ISSUE THREAD

**Issue by cwilling** [commontk/PythonQt#51]
Looks like (from README) latest code is based on r403 at sourceforge repo. I see they have just released a version 3.2 which seems to include up to r455.

Is there any plan to update code here to include the other sourceforge commits (r404 - r455)?

**Comment by jcfr** [comment]
Thanks for the report.

Branch `patched-7` is more recent (include up to r443), I changed the default branch to reference this one.

We could create patched-8 based on the latest release, do you have time to do so ?

**Comment by cwilling** [comment]
I just made a pull request (#52) consisting of just the bare patches from r445 to r455 (r444 seemed to have already been included), which could be the basis of a new patched-8 branch.

The patches all applied cleanly but unfortunately there's still a lot of work to do to make it compile on my system. That's not the fault of the new patches though - I can't compile the new default patched-7 branch either. Since the new upstream release (which includes up to r455) builds OK here, I guess the problem lies with the cmake stuff which I'm not really conversant with. Hopefully someone more knowledgeable can work on that.

**Comment by cwilling** [comment]
I resolved the build problems I had with patched-7 and the proposed patched-8 - there were two things to fix:
1. With both qt5 & qt4 installed and without specifying a Qt5_DIR, ccmake gives a choice between 4 & 5 but this doesn't work for patched-8 because there is no generated_cpp_4* directory, which cmake demands even when version 5 is selected. I needed to run (c)cmake with -DQt5_DIR=/path/to/qt5/cmake to progress past that.
2. CMakeLists.txt only adds qt5 WebkitWidgets for qt versions greater than 4 and less than 5.6.0. I think that upper limit is wrong (definitely wrong for 5.7.1) and should be removed. I've just added that change to my pull request for a new patched-8 branch. It also applies to and allows me to build patched-7.

I notice that upstream's latest release 3.2 has no generated files for any qt4 version i.e. no support for qt4. Not having to deal with both qt45 and qt5 could simplify CMakeLists.txt somewhat (but that's a bigger job for someone who understands cmake a bit better than I do).
